CMakeLists.txt

```diff
@@ -233,3 +233,20 @@ if (F3D_BUILD_APPLICATION)
   install(FILES doc/THIRD_PARTY_LICENSES.md
     DESTINATION ${F3D_LIC_DIR} COMPONENT licenses)
 endif ()
+
+# Check LFS data is available
+# XXX: In the unlikely case where data is available but git lfs is not a
+# valid command this will prevent configuration with testing
+execute_process (
+  WORKING_DIRECTORY ${F3D_SOURCE_DIR}
+  COMMAND git lfs ls-files
```

Comment:
what happens if the command fails?

Reply:
documented above, configure will fail because result var will be empty as it ouput to stderr, not stdout.